frontend/src/components/EventCalendar.tsx

```diff
@@ -42,65 +36,167 @@ const customMessages: Messages = {
     agenda: 'Agenda',
 };
 
-async function getSelectedJobAttendees(): Promise<any> {
-    const attendees = await getJobAttendees();
-    return attendees;
-}
+async function getCalendarEventsLocal(token: string) {
+    const events = await getCalendarEvents(token);
+    const jobs = await getCalendarJobs(token);
 
-async function getCurrentFamilyMembers(): Promise<any> {
-    const currentFamilyMembers = await getFamilyMembers();
-    return currentFamilyMembers;
+    let calendarEvents: Array<Job | Event> = [];
+    if (events && jobs) {
+        calendarEvents = calendarEvents.concat(events);
+        calendarEvents = calendarEvents.concat(jobs);
+    }
+    return calendarEvents;
 }
 
-export default function EventCalendar(props: EventCalendarProps) {
+export default function EventCalendar() {
     const { state } = useContext(UserContext);
     const { onClose: onViewEventClose, isOpen: isViewEventOpen, onOpen: onViewEventOpen } = useDisclosure();
     const { onClose: onSignUpClose, isOpen: isSignUpOpen, onOpen: onSignUpOpen } = useDisclosure();
-    const [selectedEvent, setSelectedEvent] = useState<any>();
-    const [eventAttendees, setAttendees] = useState<any>();
+    const [selectedEvent, setSelectedEvent] = useState<Event | Job>();
     const [familyMembers, setFamilyMembers] = useState<any>();
+    const [calendarEvents, setCalendarEvents] = useState<Array<Job | Event>>([]);
+
+    function isEvent(event: Event | Job): event is Event {
+        if ((event as Event).eventType) {
+            return true;
+        }
+        // else, its a Job
+        return false;
+    }
+
+    function deleteEventWasSuccessful(calendarEvent: DeletedEvent | ErrorResponse): calendarEvent is DeletedEvent {

```

Comment:
there are simpler checks for this by looking for the reason field on the response - no need to make it complicated